src/mapping/contentful/two-grid.tsx

```diff
@@ -9,19 +9,11 @@ export const TwoGridComponent = (props: DefaultQueryParameters) => {
     const data = TwoGridGql(props);
     const componentData: ComponentTwoGridFieldsFragment | null | undefined = data.data?.twoGrid;
 
-    const expertAdvicesProps:any = componentData?.componentLeft;
```

Comment:
Why is this removed? The components are now not filled in with data?

Reply:
My IDE showed it wasn't being used. I think it was wrong. I'll add it back.